dll/cpl/sysdm/general.c

```diff
@@ -175,7 +180,7 @@ LRESULT CALLBACK RosImageProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam
                     hCreditsBitmap = CreateBitmap(pImgInfo->cxSource, (2 * pImgInfo->cySource) + iDevsHeight + 1, pImgInfo->iPlanes, pImgInfo->iBits, NULL);
```

Comment:
Since hCreditsBitmap is "static", what would happen here if hCreditsBitmap is not NULL?

Reply:
Correct, the deleteObject call has to be protected in case the static variable remains NULL.